app.organicmaps.desktop.yml

```diff
@@ -16,7 +16,7 @@ modules:
     buildsystem: cmake-ninja
     builddir: true
     config-opts:
-      - -DCMAKE_BUILD_TYPE=Release
+      - -DCMAKE_BUILD_TYPE=Debug
```

Comment:
Debug build will work 10x times slower than Release. What problem are you trying to solve?

Reply:
https://github.com/flathub/app.organicmaps.desktop/issues/45

Reply:
> Debug build will work 10x times slower than Release

Still I can't see the difference. What am I doing wrong?
Sure that's not intended for stable build.

Reply:
Debug build contains millions of assert checks. If it works as fast as release, then you've built a release, not a debug.

Reply:
Or it actually works as fast as release.
How do I build a debug then?

Reply:
From what I can see in the docs, I was actually build a debug:
https://github.com/organicmaps/organicmaps/blob/master/docs/INSTALL.md#more-options

Reply:
It can not work faster than release, only slower. There are many ASSERT checks that are absent in the release. So I assume that you've either built a release instead of the debug, or your release is as slow as debug for some reason (or you don't see the difference with your eye because you don't test an excessive map data processing and rendering).
Debug version should display FPS by default.

Reply:
Yes, it's FPS FPS by default in this version, one can run and make sure himself.
I compared with flatpak stable so it's not my release.

I don't think it's my eyes, but the procedure to see the difference with any eyes is welcome.

Reply:
Drag and render the map in a very dense map area and you will see a rendering speed difference with your eyes.

Reply:
Could you be more specific about the procedure - what area, what zoom, etc.?
Because I already put a lot of efforts to see any evidence of the slowness, no luck so far.